In-house patch to remove the imports of local copies of six and
use system ones instead.

This is Solaris-specific and not suitable for upstream submission.

--- urllib3-1.26.15/dummyserver/handlers.py
+++ urllib3-1.26.15/dummyserver/handlers.py
@@ -14,9 +14,9 @@ from io import BytesIO
 from tornado import httputil
 from tornado.web import RequestHandler
 
-from urllib3.packages.six import binary_type, ensure_str
-from urllib3.packages.six.moves.http_client import responses
-from urllib3.packages.six.moves.urllib.parse import urlsplit
+from six import binary_type, ensure_str
+from six.moves.http_client import responses
+from six.moves.urllib.parse import urlsplit
 
 log = logging.getLogger(__name__)
 
--- urllib3-1.26.15/src/urllib3/_collections.py
+++ urllib3-1.26.15/src/urllib3/_collections.py
@@ -19,8 +19,8 @@ except ImportError:  # Platform-specific
 from collections import OrderedDict
 
 from .exceptions import InvalidHeader
-from .packages import six
-from .packages.six import iterkeys, itervalues
+import six
+from six import iterkeys, itervalues
 
 __all__ = ["RecentlyUsedContainer", "HTTPHeaderDict"]
 
--- urllib3-1.26.15/src/urllib3/connection.py
+++ urllib3-1.26.15/src/urllib3/connection.py
@@ -9,9 +9,9 @@ import warnings
 from socket import error as SocketError
 from socket import timeout as SocketTimeout
 
-from .packages import six
-from .packages.six.moves.http_client import HTTPConnection as _HTTPConnection
-from .packages.six.moves.http_client import HTTPException  # noqa: F401
+import six
+from six.moves.http_client import HTTPConnection as _HTTPConnection
+from six.moves.http_client import HTTPException  # noqa: F401
 from .util.proxy import create_proxy_ssl_context
 
 try:  # Compiled with SSL?
--- urllib3-1.26.15/src/urllib3/connectionpool.py
+++ urllib3-1.26.15/src/urllib3/connectionpool.py
@@ -35,8 +35,8 @@ from .exceptions import (
     SSLError,
     TimeoutError,
 )
-from .packages import six
-from .packages.six.moves import queue
+import six
+from six.moves import queue
 from .request import RequestMethods
 from .response import HTTPResponse
 from .util.connection import is_connection_dropped
--- urllib3-1.26.15/src/urllib3/exceptions.py
+++ urllib3-1.26.15/src/urllib3/exceptions.py
@@ -1,6 +1,6 @@
 from __future__ import absolute_import
 
-from .packages.six.moves.http_client import IncompleteRead as httplib_IncompleteRead
+from six.moves.http_client import IncompleteRead as httplib_IncompleteRead
 
 # Base Exceptions
 
--- urllib3-1.26.15/src/urllib3/fields.py
+++ urllib3-1.26.15/src/urllib3/fields.py
@@ -4,7 +4,7 @@ import email.utils
 import mimetypes
 import re
 
-from .packages import six
+import six
 
 
 def guess_content_type(filename, default="application/octet-stream"):
--- urllib3-1.26.15/src/urllib3/filepost.py
+++ urllib3-1.26.15/src/urllib3/filepost.py
@@ -6,8 +6,8 @@ import os
 from io import BytesIO
 
 from .fields import RequestField
-from .packages import six
-from .packages.six import b
+import six
+from six import b
 
 writer = codecs.lookup("utf-8")[3]
 
--- urllib3-1.26.15/src/urllib3/poolmanager.py
+++ urllib3-1.26.15/src/urllib3/poolmanager.py
@@ -13,8 +13,8 @@ from .exceptions import (
     ProxySchemeUnsupported,
     URLSchemeUnknown,
 )
-from .packages import six
-from .packages.six.moves.urllib.parse import urljoin
+import six
+from six.moves.urllib.parse import urljoin
 from .request import RequestMethods
 from .util.proxy import connection_requires_http_tunnel
 from .util.retry import Retry
--- urllib3-1.26.15/src/urllib3/request.py
+++ urllib3-1.26.15/src/urllib3/request.py
@@ -1,7 +1,7 @@
 from __future__ import absolute_import
 
 from .filepost import encode_multipart_formdata
-from .packages.six.moves.urllib.parse import urlencode
+from six.moves.urllib.parse import urlencode
 
 __all__ = ["RequestMethods"]
 
--- urllib3-1.26.15/src/urllib3/response.py
+++ urllib3-1.26.15/src/urllib3/response.py
@@ -32,7 +32,7 @@ from .exceptions import (
     ResponseNotChunked,
     SSLError,
 )
-from .packages import six
+import six
 from .util.response import is_fp_closed, is_response_to_head
 
 log = logging.getLogger(__name__)
--- urllib3-1.26.15/src/urllib3/contrib/appengine.py
+++ urllib3-1.26.15/src/urllib3/contrib/appengine.py
@@ -52,7 +52,7 @@ from ..exceptions import (
     SSLError,
     TimeoutError,
 )
-from ..packages.six.moves.urllib.parse import urljoin
+from six.moves.urllib.parse import urljoin
 from ..request import RequestMethods
 from ..response import HTTPResponse
 from ..util.retry import Retry
--- urllib3-1.26.15/src/urllib3/contrib/ntlmpool.py
+++ urllib3-1.26.15/src/urllib3/contrib/ntlmpool.py
@@ -11,7 +11,7 @@ from logging import getLogger
 from ntlm import ntlm
 
 from .. import HTTPSConnectionPool
-from ..packages.six.moves.http_client import HTTPSConnection
+from six.moves.http_client import HTTPSConnection
 
 warnings.warn(
     "The 'urllib3.contrib.ntlmpool' module is deprecated and will be removed "
--- urllib3-1.26.15/src/urllib3/contrib/pyopenssl.py
+++ urllib3-1.26.15/src/urllib3/contrib/pyopenssl.py
@@ -76,7 +76,7 @@ import sys
 import warnings
 
 from .. import util
-from ..packages import six
+import six
 from ..util.ssl_ import PROTOCOL_TLS_CLIENT
 
 warnings.warn(
--- urllib3-1.26.15/src/urllib3/contrib/_securetransport/bindings.py
+++ urllib3-1.26.15/src/urllib3/contrib/_securetransport/bindings.py
@@ -48,7 +48,7 @@ from ctypes import (
 )
 from ctypes.util import find_library
 
-from ...packages.six import raise_from
+from six import raise_from
 
 if platform.system() != "Darwin":
     raise ImportError("Only macOS is supported")
--- urllib3-1.26.15/src/urllib3/util/connection.py
+++ urllib3-1.26.15/src/urllib3/util/connection.py
@@ -4,7 +4,7 @@ import socket
 
 from ..contrib import _appengine_environ
 from ..exceptions import LocationParseError
-from ..packages import six
+import six
 from .wait import NoWayToWaitForSocketError, wait_for_read
 
 
--- urllib3-1.26.15/src/urllib3/util/queue.py
+++ urllib3-1.26.15/src/urllib3/util/queue.py
@@ -1,7 +1,7 @@
 import collections
 
-from ..packages import six
-from ..packages.six.moves import queue
+import six
+from six.moves import queue
 
 if six.PY2:
     # Queue is imported for side effects on MS Windows. See issue #229.
--- urllib3-1.26.15/src/urllib3/util/request.py
+++ urllib3-1.26.15/src/urllib3/util/request.py
@@ -3,7 +3,7 @@ from __future__ import absolute_import
 from base64 import b64encode
 
 from ..exceptions import UnrewindableBodyError
-from ..packages.six import b, integer_types
+from six import b, integer_types
 
 # Pass as a value within ``headers`` to skip
 # emitting some HTTP headers that are added automatically.
--- urllib3-1.26.15/src/urllib3/util/response.py
+++ urllib3-1.26.15/src/urllib3/util/response.py
@@ -3,7 +3,7 @@ from __future__ import absolute_import
 from email.errors import MultipartInvariantViolationDefect, StartBoundaryNotFoundDefect
 
 from ..exceptions import HeaderParsingError
-from ..packages.six.moves import http_client as httplib
+from six.moves import http_client as httplib
 
 
 def is_fp_closed(obj):
--- urllib3-1.26.15/src/urllib3/util/retry.py
+++ urllib3-1.26.15/src/urllib3/util/retry.py
@@ -17,7 +17,7 @@ from ..exceptions import (
     ReadTimeoutError,
     ResponseError,
 )
-from ..packages import six
+import six
 
 log = logging.getLogger(__name__)
 
--- urllib3-1.26.15/src/urllib3/util/ssl_.py
+++ urllib3-1.26.15/src/urllib3/util/ssl_.py
@@ -13,7 +13,7 @@ from ..exceptions import (
     SNIMissingWarning,
     SSLError,
 )
-from ..packages import six
+import six
 from .url import BRACELESS_IPV6_ADDRZ_RE, IPV4_RE
 
 SSLContext = None
--- urllib3-1.26.15/src/urllib3/util/ssltransport.py
+++ urllib3-1.26.15/src/urllib3/util/ssltransport.py
@@ -3,7 +3,7 @@ import socket
 import ssl
 
 from ..exceptions import ProxySchemeUnsupported
-from ..packages import six
+import six
 
 SSL_BLOCKSIZE = 16384
 
--- urllib3-1.26.15/src/urllib3/util/url.py
+++ urllib3-1.26.15/src/urllib3/util/url.py
@@ -4,7 +4,7 @@ import re
 from collections import namedtuple
 
 from ..exceptions import LocationParseError
-from ..packages import six
+import six
 
 url_attrs = ["scheme", "auth", "host", "port", "path", "query", "fragment"]
 
--- urllib3-1.26.15/test/__init__.py
+++ urllib3-1.26.15/test/__init__.py
@@ -18,7 +18,7 @@ except ImportError:
 
 from urllib3 import util
 from urllib3.exceptions import HTTPWarning
-from urllib3.packages import six
+import six
 from urllib3.util import ssl_
 
 try:
--- urllib3-1.26.15/test/test_collections.py
+++ urllib3-1.26.15/test/test_collections.py
@@ -3,7 +3,7 @@ import pytest
 from urllib3._collections import HTTPHeaderDict
 from urllib3._collections import RecentlyUsedContainer as Container
 from urllib3.exceptions import InvalidHeader
-from urllib3.packages import six
+import six
 
 xrange = six.moves.xrange
 
--- urllib3-1.26.15/test/test_compatibility.py
+++ urllib3-1.26.15/test/test_compatibility.py
@@ -3,7 +3,7 @@ import warnings
 import pytest
 
 from urllib3.connection import HTTPConnection
-from urllib3.packages.six.moves import http_cookiejar, urllib
+from six.moves import http_cookiejar, urllib
 from urllib3.response import HTTPResponse
 
 
--- urllib3-1.26.15/test/test_connectionpool.py
+++ urllib3-1.26.15/test/test_connectionpool.py
@@ -26,9 +26,9 @@ from urllib3.exceptions import (
     SSLError,
     TimeoutError,
 )
-from urllib3.packages.six.moves import http_client as httplib
-from urllib3.packages.six.moves.http_client import HTTPException
-from urllib3.packages.six.moves.queue import Empty
+from six.moves import http_client as httplib
+from six.moves.http_client import HTTPException
+from six.moves.queue import Empty
 from urllib3.response import HTTPResponse
 from urllib3.util.ssl_match_hostname import CertificateError
 from urllib3.util.timeout import Timeout
--- urllib3-1.26.15/test/test_fields.py
+++ urllib3-1.26.15/test/test_fields.py
@@ -1,7 +1,7 @@
 import pytest
 
 from urllib3.fields import RequestField, format_header_param_rfc2231, guess_content_type
-from urllib3.packages.six import u
+from six import u
 
 
 class TestRequestField(object):
--- urllib3-1.26.15/test/test_filepost.py
+++ urllib3-1.26.15/test/test_filepost.py
@@ -2,7 +2,7 @@ import pytest
 
 from urllib3.fields import RequestField
 from urllib3.filepost import encode_multipart_formdata, iter_fields
-from urllib3.packages.six import b, u
+from six import b, u
 
 BOUNDARY = "!! test boundary !!"
 
--- urllib3-1.26.15/test/test_queue_monkeypatch.py
+++ urllib3-1.26.15/test/test_queue_monkeypatch.py
@@ -5,7 +5,7 @@ import pytest
 
 from urllib3 import HTTPConnectionPool
 from urllib3.exceptions import EmptyPoolError
-from urllib3.packages.six.moves import queue
+from six.moves import queue
 
 
 class BadError(Exception):
--- urllib3-1.26.15/test/test_response.py
+++ urllib3-1.26.15/test/test_response.py
@@ -24,7 +24,7 @@ from urllib3.exceptions import (
     SSLError,
     httplib_IncompleteRead,
 )
-from urllib3.packages.six.moves import http_client as httplib
+from six.moves import http_client as httplib
 from urllib3.response import HTTPResponse, brotli
 from urllib3.util.response import is_fp_closed
 from urllib3.util.retry import RequestHistory, Retry
--- urllib3-1.26.15/test/test_retry.py
+++ urllib3-1.26.15/test/test_retry.py
@@ -11,8 +11,8 @@ from urllib3.exceptions import (
     ResponseError,
     SSLError,
 )
-from urllib3.packages import six
-from urllib3.packages.six.moves import xrange
+import six
+from six.moves import xrange
 from urllib3.response import HTTPResponse
 from urllib3.util.retry import RequestHistory, Retry
 
--- urllib3-1.26.15/test/test_retry_deprecated.py
+++ urllib3-1.26.15/test/test_retry_deprecated.py
@@ -12,8 +12,8 @@ from urllib3.exceptions import (
     ResponseError,
     SSLError,
 )
-from urllib3.packages import six
-from urllib3.packages.six.moves import xrange
+import six
+from six.moves import xrange
 from urllib3.response import HTTPResponse
 from urllib3.util.retry import RequestHistory, Retry
 
--- urllib3-1.26.15/test/test_util.py
+++ urllib3-1.26.15/test/test_util.py
@@ -19,7 +19,7 @@ from urllib3.exceptions import (
     TimeoutStateError,
     UnrewindableBodyError,
 )
-from urllib3.packages import six
+import six
 from urllib3.poolmanager import ProxyConfig
 from urllib3.util import is_fp_closed
 from urllib3.util.connection import _has_ipv6, allowed_gai_family, create_connection
--- urllib3-1.26.15/test/with_dummyserver/test_connectionpool.py
+++ urllib3-1.26.15/test/with_dummyserver/test_connectionpool.py
@@ -30,8 +30,8 @@ from urllib3.exceptions import (
     ReadTimeoutError,
     UnrewindableBodyError,
 )
-from urllib3.packages.six import b, u
-from urllib3.packages.six.moves.urllib.parse import urlencode
+from six import b, u
+from six.moves.urllib.parse import urlencode
 from urllib3.util import SKIP_HEADER, SKIPPABLE_HEADERS
 from urllib3.util.retry import RequestHistory, Retry
 from urllib3.util.timeout import Timeout
--- urllib3-1.26.15/test/with_dummyserver/test_https.py
+++ urllib3-1.26.15/test/with_dummyserver/test_https.py
@@ -41,7 +41,7 @@ from urllib3.exceptions import (
     SSLError,
     SystemTimeWarning,
 )
-from urllib3.packages import six
+import six
 from urllib3.util.timeout import Timeout
 
 from .. import has_alpn
--- urllib3-1.26.15/test/with_dummyserver/test_socketlevel.py
+++ urllib3-1.26.15/test/with_dummyserver/test_socketlevel.py
@@ -17,7 +17,7 @@ from urllib3.exceptions import (
     ReadTimeoutError,
     SSLError,
 )
-from urllib3.packages.six.moves import http_client as httplib
+from six.moves import http_client as httplib
 from urllib3.poolmanager import proxy_from_url
 from urllib3.util import ssl_, ssl_wrap_socket
 from urllib3.util.retry import Retry
--- urllib3-1.26.15/setup.py
+++ urllib3-1.26.15/setup.py
@@ -82,6 +82,7 @@ setup(
     ],
     package_dir={"": "src"},
     requires=[],
+    install_requires=["six"],
     python_requires=">=2.7, !=3.0.*, !=3.1.*, !=3.2.*, !=3.3.*, !=3.4.*, !=3.5.*",
     extras_require={
         "brotli": [
